users.go

```diff
@@ -34,6 +35,14 @@ func (u *UserService) ListUsers(params map[string]string) ([]*User, *ListUsersRe
 
 // CreateUser creates an awx User.
 func (u *UserService) CreateUser(data map[string]interface{}, params map[string]string) (*User, error) {
+	mandatoryFields = []string{"username", "password", "first_name", "last_name", "email"}
```

Comment:
ONLY the username is required, see https://docs.ansible.com/ansible-tower/3.2.2/html/towerapi/users.html#reviewing-the-options-endpoint